lockFileLint.js

```diff
@@ -1,52 +0,0 @@
-/*
```

Comment:
@Francois-Esquire why is this deleted?

Reply:
@JamesSingleton This was deleted since it's not used anymore I think - all the `package-lock.json` files inside the repository packages have been replaced with one single `yarn.lock`